src/app/pages/storage/storage.page.scss

```diff
@@ -1,7 +1,41 @@
-ion-thumbnail {
-  --size: 25vw;
+
+.img-card {
+  margin: calc(var(--ion-padding, 9px) / 3);
+  width: 30vw;
+
+}
+.Title {
+  color: #000;
+}
+.setTitle {
+  display: flex;
+  flex-direction: row;
+  justify-self: start;
+  padding-bottom: 10px;
+  color: #000;
+  margin: calc(var(--ion-padding, 9px) / 3);
+}
+
+.img-thumbnail {
+  --size: calc(90vw/3);
 }
 
-ion-card {
-  margin: calc(var(--ion-padding, 16px) / 4);
+.select-thumbnail {
+  --size: 90vw;
+  height: 60vw;
+}
+
+.date-card-title {
+text-align: start;
+color: #000;
+margin-left: 5vw;
+padding-bottom: 10px;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/numbersprotocol/capture-lite/pullRequest?prid=6209007)